client.py

```diff
@@ -1,26 +1,47 @@
 from __future__ import print_function
-from pprint import pprint
 import argparse
-import xmlrpclib
+import json
 
 SERVERS = {
     'brew': 'https://brewhub.engineering.redhat.com/brewhub',
-    'localhost': 'http://localhost:5000/xmlrpc',
+    'localhost': 'http://localhost:5000/api/v1.0',
 }
 
 parser = argparse.ArgumentParser()
 parser.add_argument('server', choices=SERVERS.keys())
+parser.add_argument('--product', default='RHEL-6-Server-EXTRAS-6')
```

Comment:
really great argparse changes here. Mind breaking this into a separate commit?

Reply:
I removed it from this PR and will create new PR later just for changes in `client.py`.